### PR TITLE
DrainageSystem-1 update

### PR DESCRIPTION
**Pull request by larswik**
Added IFC-file to DrainageSystem-1

## Unit test

Created an IFC file for the unit test DrainageSystem-1. See readme and IFC-file for further information.

## Covered topics

- [x] project setup
- [ ] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [x] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [x] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: DrainageSystem-1/Readme.md [hunk 1]
```diff
@@ -55,10 +55,11 @@ Network topology using `IfcDistributionPort` and `IfcRelConnectsPorts`.
 Following files correspond to this scenario:
 
 | Filename                 | Description                                                  |
-|:------------------------ | ------------------------------------------------------------ |
+| :----------------------- | ------------------------------------------------------------ |
 | `DrainageSystem-1-1.xml` | Inframodel/LandXML: PipeNetwork with Structs and Pipes       |
 | `DrainageSystem-1-2.xml` | Inframodel/LandXML: Top surface of the road and slopes/ditches |
 | `DrainageSystem-1-3.jpg` | Inframodel/LandXML: Screenshot plan view                     |
 | `DrainageSystem-1-4.jpg` | Inframodel/LandXML: Screenshot 3d view                       |
+| DrainageSystem-1-1.ifc   | IFC file implementing the drainage system according to the above content description<br />except that the IfcRoad was not decomposed into IfcFacilityPart.<br />The physical objects have both an Axis and a Body representation. |
```
Comment (pjanck) [inline DrainageSystem-1/Readme.md]: Please don't use html tags in text. Let the UI break the lines for you.

File: DrainageSystem-1/Readme.md [hunk 2]
```diff
@@ -55,10 +55,11 @@ Network topology using `IfcDistributionPort` and `IfcRelConnectsPorts`.
 Following files correspond to this scenario:
 
 | Filename                 | Description                                                  |
-|:------------------------ | ------------------------------------------------------------ |
+| :----------------------- | ------------------------------------------------------------ |
```
Comment (pjanck) [inline DrainageSystem-1/Readme.md]: Why this change? I don't think that this is valid markdown...

**Review by pjanck — CHANGES_REQUESTED** [review]
Didn't look at the IFC content, only the readme.

File: DrainageSystem-1/Readme.md (hunk 2 above)
Comment (larswik) [reply DrainageSystem-1/Readme.md]: Hmmm, I use Typora for editing markdown, and sometimes things happen. Creating a new version.

File: DrainageSystem-1/Readme.md (hunk 1 above)
Comment (larswik) [reply DrainageSystem-1/Readme.md]: Should be fixed.

Comment (larswik) [reply DrainageSystem-1/Readme.md]: So, as you see, it doesn't always look so good when letting the UI do the breaks...

File: DrainageSystem-1/Readme.md (hunk 2 above)
Comment (pjanck) [reply DrainageSystem-1/Readme.md]: Hm, if it's Typora it should be fine. I just haven't seen such syntax yet. May be fine.

**Comment by LeeGregory12d** [comment]
The IFC 4x3  file appears correct when read into 12d Model
DrainageSystem-1-in-12d-Model
